nservicebus/hosting/docker-host/index.md

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
This file is missing the standard "one space after a header, two spaces after a section" that we have for docs files.